site/src/Tour.jsx

```diff
@@ -148,14 +204,27 @@ function Tour({ run, modeName, setMapEntity }) {
           setStepIndex(nextStepIndex);
         }, 100);
       } else if (
-        (event.index === 6) &
+        (event.index === 8) &
         (event.lifecycle === LIFECYCLE.COMPLETE)
       ) {
         setMapEntity({});
         setStepIndex(nextStepIndex);
+      } else if (
+        (event.index === 9) &
```

Comment:
Hoo boy, there are a LOT of seemingly magical numbers being thrown around here- it's looking like maintaining /extending the tour might be getting a little unwieldy. Possible target for improvements in the future! 

If you find a particular area of the code is 'hot' i.e. you're constantly going back to it to make changes (and frankly, the tour seems like a likely candidate) it can be a good idea to invest some time into making it easier to extend and modify.

Reply:
This section is always a pain. It does all the handling of going from one tour step to another (opening the panel, theme selector, etc). I am able to get rid of the magic numbers, but the rest of the conditions are more necessary and take some time to understand. I'll do my best to add some comments explaining the process